README.md

```diff
@@ -226,6 +226,13 @@ ALTER TABLE users
 ADD COLUMN created_at timestamp GENERATED ALWAYS AS (id::timestamp) STORED;
 ```
 
+Cast timestamp to [ulid][]:
```

Comment:
Let's talk about the caveat of `...0000...` being added here.

Reply:
I've clarified, see if is looks good